satpy/demo/__init__.py

```diff
@@ -50,8 +50,9 @@
 
 """
 
-from .abi_l1b import get_hurricane_florence_abi, get_us_midlatitude_cyclone_abi  # noqa
-from .ahi_hsd import download_typhoon_surigae_ahi  # noqa
-from .viirs_sdr import get_viirs_sdr_20170128_1229  # noqa
-from .fci import download_fci_test_data  # noqa
-from .seviri_hrit import download_seviri_hrit_20180228_1500  # noqa
+from .abi_l1b import get_hurricane_florence_abi  # noqa: F401
+from .abi_l1b import get_us_midlatitude_cyclone_abi  # noqa: F401
```

Comment:
Why did this change from one to two lines for importing two names from one module?

Reply:
might be a line length thing